bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt

```diff
@@ -23,5 +23,8 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
+<%- if config[:ext] == 'rust' -%>
+        rubygems: 'latest' # for best Rust extension support
```

Comment:
We should use https://github.com/oxidize-rb/actions/blob/main/setup-ruby-and-rust/readme.md for CI on rust if possible. It’s well supported and this current configuration will probably not work in all scenarios, it also will be slow due to no caching etc

Reply:
@ianks there is only v1 tag and that doesn't support needed `rubygems` option. Would it be possible to tag new version?